frontend/src/renderer/components/Charts/AtomicHeatMap.svelte

```diff
@@ -0,0 +1,131 @@
+<script>
+    import { chart } from "svelte-apexcharts";
+
+    // All the data included from the backend, will be parsed here
+    export let data;
+
+    // Parse the needed data for the heatmap
+    function getData() {
+        let formattedData = [];
+        let tempData = [];
+        for (let i = 0; i < data.length; i++) {
+            tempData[i] = [];
+            for (let j = 0; j < data[i].length; j++) {
+                tempData[i][j] = {
+                    y: data[i][j].toFixed(2),
+                    x: String.fromCharCode(j + 65)
+                };
+            }
+            formattedData.push({
+                name: i,
+                data: tempData[i]
+            });
+        }
+
+        return formattedData;
+    }
+
+    const colors = [
+        "#0a9400",
+        "#199a00",
+        "#2a9f00",
+        "#3ca500",
+        "#4eab00",
+        "#62b000",
+        "#77b600",
+        "#8dbb00",
+        "#a4c100",
+        "#bdc700",
+        "#ccc300",
+        "#d2b300",
+        "#d8a300",
+        "#dd9200",
+        "#e37f00",
+        "#e86c00",
+        "#ee5700",
+        "#f44100",
+        "#f92a00",
+        "#ff1200"
+    ];
+
+    function ranges() {
+        const result = [];
+        for (let i = 0; i < 20; i++) {
+            result.push({
+                from: i/2,
+                to: i/2 + .5,
+                color: colors[i]
+            });
+        }
+        result[result.length - 1].to = Number.MAX_SAFE_INTEGER;
+        return result;
+    }
+
+    const options = {
```

Comment:
The docs for this are actually great! Highly recommend ApexCharts for future graphs/charts (not to be confused with Apex the housing development next to RIT which I do not recommend)

Reply:
Yeah glad we found an alternative to Carbon charts, that documentation sucked